use-timescale/hypertables/improve-query-performance.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Improve query performance
+excerpt: How and why to use chunk skipping to optimize hypertable performance
+products: [cloud, mst, self_hosted]
+keywords: [hypertables, indexes, chunks]
+---
+
+# Improve query performance
```

Comment:
You are very right in this :-) Iain suggested this page title to be "Improve query performance" because we plan on adding more optimization stuff in there - eventually making it into a section, I think. However, changing titles back and forth won't be good for SEO or doc user experience, that's why we put in this more general title from the very beginning.

Reply:
What I  was thinking about was this one being just one of **multiple pages** under a larger section on the docs called "Improved Query Performance", not as a section on a larger page (cause it already feels like it has a lot and we can add way more following my comment above).

But if you think that the way to go is one page for all, I am not going to object - you know better about how to optimally structure our documentation!

Reply:
Yeah, I also meant making it into a section with multiple pages, not a section on the page. We'll get there :-) Thank you!